tests/rpc/rpc_05-state_test.go

```diff
@@ -31,15 +33,22 @@ func TestStateRPCResponseValidation(t *testing.T) { //nolint:tparallel
 	getBlockHashCancel()
 	require.NoError(t, err)
 
+	// TODO: Improve runtime tests
+	// https://github.com/ChainSafe/gossamer/issues/3234
```

Comment:
```suggestion
	// TODO: Improve runtime tests #3234
```


could do the similar thing at other places as well

Reply:
hmm, but why?